test/run_tests.cpp

```diff
@@ -1925,10 +1925,10 @@ static void run_test(TestCase *test_case) {
 
     if (test_case->is_parseh) {
         if (buf_len(&zig_stderr) > 0) {
-            printf("\nparseh emitted warnings:\n");
+            printf("\n!!!!! parseh emitted warnings:\n");
```

Comment:
it has to do with built-in types in GCC, that aren't defined in any file. also, it seems that parseh does not take into account unions and they are marked as a complex types. 
and some some macros needs to be defined as they are when you execute the real C compiler.